instances/molecularEntity/2-Bromo-2-chloro-1,1,1-trifluoroethane.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/2-Bromo-2-chloro-1,1,1-trifluoroethane",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "2-Bromo-2-chloro-1,1,1-trifluoroethane is a nonflammable, halogenated, hydrocarbon anesthetic that provides relatively rapid induction with little or no excitement. ",
```

Comment:
please delete the unnecessary space at the end of the definition and add the source of the definition (e.g. as citation and/or link to the source)